spharpy/classes/coordinates.py

```diff
@@ -501,6 +590,27 @@ def _check_weights(self, weights):
 
         return weights
 
+    def _check_quadrature(self):
+        r"""Check if the sampling is a are valid quadrature.
+
+        Returns
+        -------
+        check : bool
+            Flag which indicates if quadrature is a valid quadrature
+        """
+        if self.n_max is None or self.weights is None:
```

Comment:
Not very likely to happen, but we can't check this way for zeroth order grids:
```suggestion
        if self.n_max in [None, 0] or self.weights is None:
```

Reply:
Are we sure a 0-order grid isn't a quadrature, at least in the mathematical sense? If so, we need need to change something in the gaussian grid ( if n_max == 0, quadrature = false or so).

Reply:
True, it would be in the mathematical sense. Probably useless in practice - but I agree to leave everything as is to be clean.